data/mock_data/generate_mock_data.sh

```diff
@@ -0,0 +1,4 @@
+set -uex
+touch mock_read.R1.fastq.gz
+touch mock_read.R2.fastq.gz
+touch mock_ref_seq.gbk
```

Comment:
Since we're planning to put this inside the `data/mock_data` folder I guess we could avoid using the `mock` in the naming scheme.

Also, please add more files for the same.

```suggestion
touch 001_R1.fastq.gz
touch 001_R2.fastq.gz
touch ref.gbk
touch ref.fasta
```

Reply:
Ok! I'll add some more files and commit your suggestion